syncd/FlexCounter.h

```diff
@@ -13,9 +13,14 @@ extern "C" {
 #include <condition_variable>
 #include <unordered_map>
 #include <memory>
+#include <utility>
 
 namespace syncd
 {
+    typedef std::pair<sai_object_id_t, sai_object_id_t> gearboxPortPair;
+#define vidSystemSide std::get<0>
+#define vidLineSide   std::get<1>
```

Comment:
move this out of namespace, defines are not scoped by namespaces

Reply:
Move it to FlexCounter.cpp